connector/src/parsers/votes.ts

```diff
@@ -0,0 +1,23 @@
+import { QueryResult } from '@aragon/connect-thegraph'
+import Vote, { VoteData } from '../entities/Vote'
+
+export function parseVotes(
+  result: QueryResult,
+  connector: any
+): Vote[] {
+  const votes = result.data.votes
+
+  if (!votes) {
+    throw new Error('Unable to parse votes.')
+  }
+
+  const datas = votes.map(
+    (vote: any): VoteData => {
```

Comment:
In  all parsers where we are using `any` we should use the generated types. I see we didn't generate them actually. 
To do so you can go to https://graphql-code-generator.com/  and paste the subgraph.schema and on the most right section will be the generated types.

Reply:
allmost all the types says that are unknown , like `ID` , `Bytes` etc.i guess that is because those are theGraph types

Reply:
We decided to remove the generated type in the end as the don't add that much value. As we explore better ways to implement connectors we can consider it again.